alacritty/src/renderer/mod.rs

```diff
@@ -411,12 +411,12 @@ struct InstanceData {
     g: f32,
     b: f32,
     // Background color.
-    bg_r: f32,
-    bg_g: f32,
-    bg_b: f32,
-    bg_a: f32,
+    bg_r: u8,
+    bg_g: u8,
+    bg_b: u8,
+    bg_a: u8,
     // Flag indicating that glyph uses multiple colors, like an Emoji.
-    multicolor: u8,
+    multicolor: i8,
```

Comment:
why you've changed u8 to i8 here, seems arbitrary.

Reply:
Because it's always been an i8 in the shader. And even though it might not matter, we should have consistent datatypes.